### ISSUE TITLE
Question: How can I update my configuration without deleting profile?

### ISSUE THREAD

**Issue by mrtnvgr** [schizofox/schizofox#54]

**Comment by sioodmy** [comment]
posibly related to #47

**Comment by mrtnvgr** [comment]
> posibly related to #47 

No, I just wonder how can I update extensions, user*.css and everything, without deleting the profile.

**Comment by sioodmy** [comment]
I don't think its currently possible, but we should work on it in the future

**Comment by mrtnvgr** [comment]
Ok!

**Comment by mrtnvgr** [comment]
> I don't think its currently possible, but we should work on it in the future

So if for example uBlock Origin will release a new version, system update/rebuild will cause breakage?

**Comment by sioodmy** [comment]
Technically we could use built-in firefox addon auto-update feature. I'm also thinking about creating separate repo with collection of addons built from source

**Comment by NotAShelf** [comment]
The problem with built addons is that they will be unsigned. In case some one wants signed addons only (a valid request) they will not be able to use the repo.

**Comment by mrtnvgr** [comment]
> The problem with built addons is that they will be unsigned. In case some one wants signed addons only (a valid request) they will not be able to use the repo.

FDroid signs their packages with their own keys, maybe you can do something similar?

**Comment by NotAShelf** [comment]
I never looked into signing firefox extensions, but I'll take a look.

**Comment by sioodmy** [comment]
that would require patching ff source code (I think). we are not ready for that

**Comment by sioodmy** [comment]
and that wouldnt be possible for dark reader, because its built on user side
